packages/tailscale/tailscale.pacscript

```diff
@@ -0,0 +1,36 @@
+name="tailscale"
+pkgver="1.54.0"
+url="https://github.com/tailscale/tailscale/archive/refs/tags/v${pkgver}.tar.gz"
+homepage="https://tailscale.com"
+depends=('iptables')
+makedepends=('golang-go' 'git')
+pkgdesc="Private WireGuard networks made easy"
+maintainer="Nathan <ndowens@artixlinux.org>"
+arch=('amd64')
+repology=("project: ${name}")
+incompatible=('ubuntu:latest' 'debian:stable')
```

Comment:
`ubuntu:latest` and `debian:stable` are identifiers for the GitHub builders themselves, the actual versions of those right now are `ubuntu:jammy` and `debian:bookworm`, you should change the array to these values.